pennylane/ops/qubit/non_parametric_ops.py

```diff
@@ -1610,13 +1609,13 @@ def num_params(self):
     # pylint: disable=unused-argument
     @staticmethod
     def compute_matrix(
-        control_wires, control_values=None, **kwargs
+        n_control_wires, control_values=None, **kwargs
     ):  # pylint: disable=arguments-differ
         """Canonical matrix representation of the MultiControlledX operator.
 
         Args:
-            control_wires (Iterable): wires to place controls on
-            control_values (str): string of bits determining the controls
+           n_control_wires (int): number of control wires
```

Comment:
how come this was renamed?

Reply:
I realised that all this jazz is not needed, just the number of control wires

Reply:
This causes an issue down below though in `MultiControlledX.adjoint()`; it is passing `n_control_wires` and `work_wires` to `__init__`, but `__init__` expects `control_wires` and `control_values`!

Reply:
Oh shoot...I thought I fixed this somewhere else? 

This gate has given me so many headaches already.